elastic/apm-server
# Remove `timeseries.instance`

## Motivation/summary

Remove this field as we are not using it, and we can use `_tsid` instead when we migrate to TSDS.

## Checklist

~- [ ] Documentation has been updated~

## How to test these changes

1. Ensure transaction metrics are recorded
2. Check that `timeseries.instance` is not indexed

## Related issues

Closes https://github.com/elastic/apm-server/issues/9018
